plugins/techdocs/mkdocs/container/techdocs-core/src/core.py

```diff
@@ -19,17 +19,24 @@
 
 from mkdocs.contrib.search import SearchPlugin
 
+from mkdocs_monorepo_plugin.plugin import MonorepoPlugin
+
 class TechDocsCore(BasePlugin):
     def on_config(self, config):
        # Theme
        config['theme'] = Theme(name="material")
 
        # Plugins
        del config['plugins']['techdocs-core']
-
+       
```

Comment:
Remember to set your editor to remove trailing whitespace!

Reply:
Added #1423 to help avoid these issues. We can probably merge this and address this in subsequent PRs, or merge that first